x/crconsent/keeper/request.go

```diff
@@ -0,0 +1,106 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/metabelarus/mbcorecr/x/crconsent/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	"strconv"
+)
+
+// GetRequestCount get the total number of request
+func (k Keeper) GetRequestCount(ctx sdk.Context) int64 {
+	store :=  prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RequestCountKey))
+	byteKey := types.KeyPrefix(types.RequestCountKey)
+	bz := store.Get(byteKey)
+
+	// Count doesn't exist: no element
+	if bz == nil {
+		return 0
+	}
+
+	// Parse bytes
+	count, err := strconv.ParseInt(string(bz), 10, 64)
+	if err != nil {
+		// Panic because the count should be always formattable to int64
+		panic("cannot decode count")
+	}
+
+	return count
+}
+
+// SetRequestCount set the total number of request
+func (k Keeper) SetRequestCount(ctx sdk.Context, count int64)  {
+	store :=  prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RequestCountKey))
+	byteKey := types.KeyPrefix(types.RequestCountKey)
+	bz := []byte(strconv.FormatInt(count, 10))
+	store.Set(byteKey, bz)
+}
+
+func (k Keeper) CreateRequest(ctx sdk.Context, msg types.MsgCreateRequest) {
+	// Create the request
+    count := k.GetRequestCount(ctx)
```

Comment:
As a next step we need to map requests to services. Ignore this comment for now.
Look at as example: x/crsign/keeper/id2record.go